src/ophys_etl/modules/segmentation/modules/feature_vector_segmentation.py

```diff
@@ -0,0 +1,41 @@
+import matplotlib
+import argschema
+import pathlib
+
+from ophys_etl.modules.segmentation.modules.schemas import \
+    FeatureVectorSegmentationInputSchema
+
+from ophys_etl.modules.segmentation.graph_utils.\
+    feature_vector_segmentation import (
+        FeatureVectorSegmenter)
+
+
+matplotlib.use('Agg')
```

Comment:
I'm not seeing matplotlib used in this module.
Incidentally, I think some of our issues with this Agg backend in the past have been because we use `matplotlib.pyplot` but everything is accessible with out it.
For example:
```
import matplotlib.pyplot as plt
fig, axes = plt.subplots(1, 2)
```
vs.
```
from matplotlib.figure import Figure
fig = Figure()
axes = [fig.add_subplot(1, 2, i) for i in [1, 2]]
```